host/3.0/buster/amd64/base.Dockerfile

```diff
@@ -18,9 +18,9 @@ COPY --from=runtime-image ["/workers", "/workers"]
 
 RUN apt-get update && \
     apt-get install -y gnupg wget unzip && \
-    wget https://functionscdn.azureedge.net/public/ExtensionBundles/Microsoft.Azure.Functions.ExtensionBundle/1.0.0/Microsoft.Azure.Functions.ExtensionBundle.1.0.0.zip && \
-    mkdir -p /FuncExtensionBundles/Microsoft.Azure.Functions.ExtensionBundle/1.0.0 && \
-    unzip /Microsoft.Azure.Functions.ExtensionBundle.1.0.0.zip -d /FuncExtensionBundles/Microsoft.Azure.Functions.ExtensionBundle/1.0.0 && \
-    rm -f /Microsoft.Azure.Functions.ExtensionBundle.1.0.0.zip
+    wget https://functionscdn.azureedge.net/public/ExtensionBundles/Microsoft.Azure.Functions.ExtensionBundle/1.1.0/Microsoft.Azure.Functions.ExtensionBundle.1.1.0.zip && \
+    mkdir -p /FuncExtensionBundles/Microsoft.Azure.Functions.ExtensionBundle/1.1.0 && \
+    unzip /Microsoft.Azure.Functions.ExtensionBundle.1.1.0.zip -d /FuncExtensionBundles/Microsoft.Azure.Functions.ExtensionBundle/1.1.0 && \
+    rm -f /Microsoft.Azure.Functions.ExtensionBundle.1.1.0.zip
```

Comment:
Just making sure, you need this updated for 3.0 too?

Reply:
i can make the change in the next 3.0 release. Not sure if 3.0 has it already

Reply:
yes